rfcs/0000-sudt-info/0000-sudt info.md

```diff
@@ -0,0 +1,184 @@
+---
+Number: "0000"
+Category: Standards Track
+Status: Proposal
+Author: Xiang Cheng
+Organization: Lay2dev
+Created: 2021-12-25
+---
+
+# sudt info
+
+## 1.Overview
+
+sudt is a very simple udt standard. With the udt deployed by this standard, we can only get the balance from the chain, but not the token name, token symbol, decimals and other parameters. 
```

Comment:
SUDT should be used instead 'sudt' everyone, to be consistent with [RFC 0025](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0025-simple-udt/0025-simple-udt.md).